server/odc-server/pom.xml

```diff
@@ -148,6 +148,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
why should we need this?

Reply:
mock static and MockVerify need this package